packages/web-components/fast-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../styles";
+import { focusVisible } from "../styles/focus";
+import { SystemColors } from "../styles/system-colors";
+import { heightNumber } from "../styles/size";
+import { forcedColorsStylesheetBehavior } from "../styles/match-media-stylesheet-behavior";
+
+export const RadioGroupStyles = css`
+    ${display("inline-flex")} :host {
+        --input-size: calc((${heightNumber} / 2) + var(--design-unit));
+        align-items: center;
+        outline: none;
+        margin: calc(var(--design-unit) * 1px) 0;
+        user-select: none;
+        position: relative;
+        flex-direction: row;
+        transition: all 0.2s ease-in-out;
+    }
+
```

Comment:
Is all this styling really necessary? It seems this control is primarily concerned with *behavior* and that styling should be left to the radio's themselves.

Reply:
Only thing I would point to here is if there is value in supporting appearance/orientation from the radiogroup itself so folks don't need to wrap themselves, add a class, etc. But agree it seems like we have a lot of styling here.

Reply:
I think there is more that can be removed here. 

1. I don't think this should be setting `--input-size`
2. This doesn't get focus, so it probalby doesn't need the outline removal
3. user-select is handled by the inputs
4. The group doesn't get foucs, so it doesn't need focus-visible styles
5. The disabled style should be handled by the individual disabled controls - we shouldn't be setting the *region* to disabled opacity

Reply:
think I did all the removals this time